bookchain-miner/src/block.hpp

```diff
@@ -1,20 +1,29 @@
 #pragma once
 #include <string>
 
-constexpr size_t kilobyte = 1 * 1024;
+constexpr size_t hashLength = 20;
+constexpr size_t hashBufferLength = hashLength + 1;  // Saving room for the null byte
+constexpr size_t megabyte = 1 * 1024 * 1024;
```

Comment:
For my own education can you explain what the `1 *` is for?

Reply:
For human reference lol. In my mind it's easier to do the math and slightly more explicit to label the byte there to get the full megabyte.
1 byte * 1024 = kilobyte;
kilobyte * 1024 = megabyte